src/CredentialsWrapper.php

```diff
@@ -136,6 +146,14 @@ public static function build(array $args = [])
         return new CredentialsWrapper($loader, $authHttpHandler);
     }
 
+    /**
+     * @return string The quota project associated with the credentials.
+     */
+    public function getQuotaProject()
+    {
+        return $this->credentialsFetcher->getQuotaProject();
```

Comment:
We should upgrade the minimal `google/auth` version to [^1.9](https://github.com/googleapis/google-auth-library-php/releases/tag/v1.9.0) so that this cannot throw a fatal error

Reply:
Or even better, add an interface check:

```php
public function getQuotaProject()
{
    if ($this->credentialsFetcher instanceof \Google\Auth\GetQuotaProjectInterface) {
        return $this->credentialsFetcher->getQuotaProject();
    }
}
```

Reply:
I can approve once this is added!

Reply:
Done!